docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,120 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with and without storage providers
+
+## Setting up etcd-druid
+
+### Clone the Etcd-Druid Github repo
+
+```sh
+# clone the repo
+git clone https://github.com/gardener/etcd-druid.git
+# cd into etcd-druid folder
+cd etcd-druid
+```
+
+---
+
+> **Note:** To setup Etcd-Druid with backups enabled on a [LocalStack](https://github.com/localstack/localstack) provider, refer [this document](https://github.com/gardener/etcd-druid/blob/master/docs/development/getting-started-locally-localstack.md)
+
+---
+
+### Setting up the Kind Cluster
```

Comment:
Can we maybe add a note here that we use a kind cluster because it's convenient, but any other k8s cluster would also suffice?